broker/scripts/session.up.sh

```diff
@@ -11,10 +11,9 @@ LOCAL_BROKER_PORT="$9"
 
 . scripts/bridge_functions.sh
 
-# Set the interface to UP state
-ip link set dev $INTERFACE up mtu $MTU
-
 # Add the interface to our bridge
 ensure_bridge digger${MTU}
-brctl addif digger${MTU} $INTERFACE
+ip link set dev $INTERFACE master digger${MTU} mtu $MTU up
 
+# Turn on bridge port isolation
+bridge link set dev $INTERFACE isolated on
```

Comment:
This looks nicer than ebtables indeed, but OTOH there is a race condition here now, is there? Between the time this is added to the bridge, and when this `bridge link set` is executed, the host *can* communicate with all other hosts?

Reply:
Thanks for pointing this out. I think you might be right. Maybe we can `up` the link only after setting `isolated on`. I'll test that.

Reply:
Would a similar "isolated on" be needed in the mtu_changed script?

Reply:
I just checked, and yes, it would be necessary to set to isolated again when changing bridges with the mtu script.  The port status can be seen in /sys/class/net/$INTERFACE/brport/isolated

Also, it's not necessary to use the bridge command.  You can also simply ```echo 1 > /sys/class/net/$INTERFACE/brport/isolated```

Reply:
Pushed a fix to my branch. Thanks for testing.